src/app/shared/components/wallet-info/wallet-info.component.html

```diff
@@ -1,4 +1,13 @@
 <div *ngIf="wallet$ | async as wallet" class="wallet-info">
+  <div class="wallet-info__item" *ngIf="faucetUrl">
+    <span class="wallet-info__value">
+    <a class="navigation__item"
```

Comment:
.navigation__item class won't work as you expected here. If you want to style this element as items in app.component.html, copy styles for .navigation__item from app.component.scss file otherwise just delete navigation__item class from a template.